<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>readMemWidget</class>
 <widget class="QWidget" name="readMemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SCC Memory Reader</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="1" column="1" rowspan="2">
    <layout class="QGridLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Node ID:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLineEdit" name="lE_StartAddr"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lE_numBytes"/>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="cB_SubDest"/>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="cB_TileID"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Start Address</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QTextBrowser" name="memWindow"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Number of Bytes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="2">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>91</width>
         <height>24</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>91</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="rB_8bit">
       <property name="text">
        <string>8 bit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rB_16bit">
       <property name="text">
        <string>16 bit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rB_32bit">
       <property name="text">
        <string>32 bit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rB_64bit">
       <property name="text">
        <string>64 bit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="3">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pB_Reload">
       <property name="text">
        <string>Reload  </string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>31</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pB_SaveAs">
       <property name="text">
        <string>Reload &amp;&amp; Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>31</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pB_Close">
       <property name="text">
        <string>  Close  </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>memWindow</tabstop>
  <tabstop>rB_8bit</tabstop>
  <tabstop>rB_16bit</tabstop>
  <tabstop>rB_32bit</tabstop>
  <tabstop>rB_64bit</tabstop>
  <tabstop>pB_Reload</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
